src/phoenix/experimental/evals/models/vertex.py

```diff
@@ -149,7 +149,29 @@ async def _completion_with_retry(**kwargs: Any) -> Any:
             response = await self._model.generate_content_async(
                 contents=prompt, generation_config=generation_config, **kwargs
             )
-            candidate = response.candidates[0]
-            return candidate.text
+            return self._parse_response_candidates(response)
 
         return await _completion_with_retry(**kwargs)
+
+    def _parse_response_candidates(self, response: Any) -> Any:
+        if hasattr(response, "candidates"):
+            if isinstance(response.candidates, list) and len(response.candidates) > 0:
+                try:
+                    candidate = response.candidates[0].text
+                except ValueError:
+                    printif(
+                        self._verbose, "The 'candidates' object does not have a 'text' attribute."
+                    )
+                    printif(self._verbose, response.candidates[0])
+                    candidate = "NOT_PARSABLE"
```

Comment:
Do we want to be hard-coding this? Isn't it more genuine to return empty string?